app.ts

```diff
@@ -1,33 +1,32 @@
-const express = require('express');
-const path = require('path');
-const nunjucks = require('nunjucks');
```

Comment:
I'm surprised the linter has asked for the `;` to be removed as normally it asks for them to be included

Reply:
Was initially using eslitn for formatting now changed to using prettier after looking at documentation
https://typescript-eslint.io/rules/semi/

Now using rule to add semicolons before lines that may cause ASI failures
https://prettier.io/docs/en/options.html#semicolons

Reply:
`semicolons before lines`?

Reply:
following the JS standard codestyle https://standardjs.com/rules.html#semicolons